collector/filesystem_linux.go

```diff
@@ -59,8 +59,15 @@ func (c *filesystemCollector) GetStats() ([]filesystemStats, error) {
 		}
 
 		var ro float64
-		if (buf.Flags & readOnly) != 0 {
-			ro = 1
+	OPTIONS:
```

Comment:
I'm not a fan of label jumps. Is there a better way to do this?

Reply:
I dislike them too, but I thought the equivalent conditional was sort of ugly.
```go
		var ro float64
		for _, option := range strings.Split(labels.options, ",") {
			if option == "ro" {
				ro = 1
				break
			}
			if option == "rw" {
				break
			}
		}
```

A switch statement seems more idiomatic but I'm happy to change styles for sake of dropping labels.

Reply:
Yes, a switch would be better.

Is there really a need to break? The efficiency gain is probably minimal.

Reply:
The only reason to `break` is a minimal efficiency gain ;-)

Dropped the branching since, to your point, we're only scanning tens of bytes.

Cheers.